src/index.spec.js

```diff
@@ -73,6 +73,22 @@ describe('desciple-core-api', () => {
       expect(verifiedAttestor).to.equal(attestorSsid)
     })
 
+    it('should be able to not verify an attestation of a revoked claim', async () => {
+      let ssid = await discipl.newSsid('memory')
+      await discipl.claim(ssid, { 'need': 'beer' })
+      let claimlink2 = await discipl.claim(ssid, { 'need': 'wine' })
+
+      let attestorSsid = await discipl.newSsid('memory')
+
+      await discipl.attest(attestorSsid, 'agree', claimlink2)
+      await discipl.revoke(ssid, claimlink2)
+
+      let verifiedAttestor = await discipl.verify('agree', claimlink2, [ssid, null, { 'did': 'did:discipl:memory:1234' }, attestorSsid])
+
+      // The first ssid that is valid and proves the attestation should be returned
```

Comment:
Comment should reflect that none of the ssids are valid.

Reply:
altered the comment